deploy-agent/deployd/__init__.py

```diff
@@ -12,9 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os
-
-# Pinterest specific settings
-IS_PINTEREST = True if os.getenv("IS_PINTEREST", "false") == "true" else False
```

Comment:
what's reason for removing this flag and related checks?

Reply:
In oder to achieve the same goal, we should parametrize this entries so they are not attached to Pinterest environment at all.
For example, better than say: "This only works for pinterest"
is to say: "Ok you need to configure X in order to get X to working, for example, cluster configs or Phobo"